chain/epoch-manager/src/validator_selection.rs

```diff
@@ -466,12 +460,13 @@ mod old_validator_selection {
         };
 
         let block_producer_proposals = order_proposals(proposals.values().cloned());
-        let (block_producers, not_block_producers, bp_stake_threshold) = select_block_producers(
+        let (block_producers, _not_block_producers, bp_stake_threshold) = select_block_producers(
             block_producer_proposals,
             max_bp_selected,
             min_stake_ratio,
             protocol_version,
         );
+        #[allow(deprecated)]
         let (chunk_producer_proposals, chunk_producers, cp_stake_threshold) =
             if ProtocolFeature::ChunkOnlyProducers.enabled(protocol_version) {
```

Comment:
Assume `ProtocolFeature::ChunkOnlyProducers.enabled(protocol_version)` is true and simplify code.